### DISCUSSION TITLE
JIRA_AUTH_TYPE is not getting used

### DISCUSSION THREAD

**Discussion by TimUntersberger** [ankitpokhrel/jira-cli#355] [General] [answered]
Hello!

First of all thank you for the amazing cli client!

My `JIRA_AUTH_TYPE` environment variable is set to `bearer`.
And my `JIRA_API_TOKEN` is also set.

I now try to run `jira init --debug` to generate the configuration file, but it fails.

Request
```
GET /rest/api/2/myself HTTP/1.1
Host: <JIRA_HOST>
Authorization: Basic <JIRA_API_TOKEN>
```

Response
```
HTTP/1.1 401 
Transfer-Encoding: chunked
Content-Security-Policy: frame-ancestors 'self'
Content-Type: text/html;charset=UTF-8
Date: Fri, 29 Apr 2022 11:17:28 GMT
Referrer-Policy: strict-origin-when-cross-origin
Server: Apache/2.4.41 (Ubuntu)
Strict-Transport-Security: max-age=86400; includeSubDomains
Strict-Transport-Security: max-age=31536000
Www-Authenticate: OAuth realm="https%3A%2F%2Fjira.cgm.ag"
X-Arequestid: 677x22817861x12
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Seraph-Loginreason: AUTHENTICATED_FAILED
X-Ua-Compatible: IE=edge,chrome=1
X-Xss-Protection: 1; mode=block
```

It seems like environment variable is not being respected.

Any help is appreciated!

**Comment by ankitpokhrel** [comment]
Hi @TimUntersberger, this feature is not released yet so you need to use the build from `main` branch. You can either [compile it locally](https://github.com/ankitpokhrel/jira-cli#development) or use following command to install (requires go1.16+).

```sh
go install github.com/ankitpokhrel/jira-cli/cmd/jira@latest 
```

If you are using Homebrew, you can do:

```sh
brew install jira-cli --HEAD
```

Related: https://github.com/ankitpokhrel/jira-cli/issues/353

**Reply by ankitpokhrel** [reply]
I will work on the new release soon, hopefully in few weeks.

**Reply by TimUntersberger** [reply]
Thank you!

**Comment by ankitpokhrel** [comment, marked as answer]
This change is available in the new release [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
